docs/markdown/foundations/content_standards.md

```diff
@@ -0,0 +1,928 @@
+---
+title: Content Design Standards
+description: Here are the nuts and bolts of writing at Pinterest—when to capitalize, what a list should look like, the right structure for an app store description and more. For more information about something that’s not covered here, reach out to contentdesign@pinterest.com.
```

Comment:
I would highly recommend against putting this email up publicly, unless they're prepared for lots of spam or absolutely need it to be public. Instead maybe link to a slack channel that is internal only?

Reply:
I wonder if we can pinch link a "mailto:" link, and then say send an email to the [content design team](url)

Reply:
If that doesn't work, and you'd like to include the email, at least space it out into something like `contentdesign (at) pinterest (dot) com` or something similar

Reply:
For the record pinch does not allow mailto links as URLs